pulp_docker/app/registry.py

```diff
@@ -42,9 +42,10 @@ class Registry(Handler):
 
         """
         accepted_media_types = []
-        for header, value in request.raw_headers:
+        for header, values in request.raw_headers:
             if header == b'Accept':
-                accepted_media_types.append(value.decode('UTF-8'))
+                values = [v.strip().decode('UTF-8') for v in values.split(b", ")]
```

Comment:
please remove the space after comma split(',')
I think normally per HTTP spec when providing multiple values in the header,each value is delimited by a comma, not sure about the presence of the space.

Reply:
```
In [15]: x='multiple, types'                                                                                                                                                                                       

In [16]: y= x.split(', ')                                                                                                                                                                                          

In [17]: y[0].strip()                                                                                                                                                                                              
Out[17]: 'multiple'

In [18]: x='multiple,types'                                                                                                                                                                                        

In [19]: z= x.split(', ')                                                                                                                                                                                          

In [20]: z[0].strip()                                                                                                                                                                                              
Out[20]: 'multiple,types'

In [21]:  
```

Reply:
tested and it works!
```
http HEAD 'http://localhost:24816/v2/foo/manifests/musl' 'Accept:application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.docker.distribution.manifest.v2+json'
HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Disposition: attachment; filename=dbab3ccf5ebe2d1c79131966766ff2156df89ed538e0c8fb9a1f087b503a65
Content-Length: 1638
Content-Type: application/vnd.docker.distribution.manifest.list.v2+json
Date: Tue, 06 Aug 2019 15:22:43 GMT
Docker-Content-Digest: sha256:2edbab3ccf5ebe2d1c79131966766ff2156df89ed538e0c8fb9a1f087b503a65
Docker-Distribution-API-Version: registry/2.0
Last-Modified: Tue, 06 Aug 2019 14:55:50 GMT
Server: Python/3.7 aiohttp/3.5.4



(pulp) [vagrant@pulp3-source-fedora29 pulp_docker]$ http HEAD 'http://localhost:24816/v2/foo/manifests/musl' 'Accept:application/vnd.docker.distribution.manifest.list.v2+json,application/vnd.docker.distribution.manifest.v2+json'
HTTP/1.1 200 OK
Accept-Ranges: bytes
Content-Disposition: attachment; filename=dbab3ccf5ebe2d1c79131966766ff2156df89ed538e0c8fb9a1f087b503a65
Content-Length: 1638
Content-Type: application/vnd.docker.distribution.manifest.list.v2+json
Date: Tue, 06 Aug 2019 15:22:50 GMT
Docker-Content-Digest: sha256:2edbab3ccf5ebe2d1c79131966766ff2156df89ed538e0c8fb9a1f087b503a65
Docker-Distribution-API-Version: registry/2.0
Last-Modified: Tue, 06 Aug 2019 14:55:50 GMT
Server: Python/3.7 aiohttp/3.5.4



(pulp) [vagrant@pulp3-source-fedora29 pulp_docker]$ 
```

Reply:
good point, updated!